docs/core/track_fitting.md

```diff
@@ -1 +1,61 @@
 # Track Fitting
+The track fitting algorithms estimate the track 
+parameters. It is part of the pattern recognition/track 
+reconstruction/tracking. We can run the 
+track fitting algorithms, 
+after we allocated all hits to single tracks with the 
+help of a track finding algorithm. It is not necessary, 
+that all points of a track are present.
+
+Currently we have implementations for three different 
+fitters:
+* Kalman Filter
+* GSF
+* Global Chi-Square Fitter (GX2F) [wip]
+?? is GSF a least square fit?
+Eventhough all of them are least-squares fits, the 
+concepts are quite different. Therefore, we should not 
+expect identical results from all of them.
+
+
+## Kalman Filter (KF)
+The Kalman Filter is an iterative fitter. It 
+successively combines measurments to obtain an 
```

Comment:
Maybe add that this is done by propagating the start parameters/integrating the EOM through the geometry